pkgs/applications/audio/spotifyd/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkgconfig, openssl
+, withALSA ? true, alsaLib ? null
+, withPulseAudio ? false, libpulseaudio ? null
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "spotifyd-${version}";
+  version = "0.2.4";
+
+  src = fetchFromGitHub {
+    owner = "Spotifyd";
+    repo = "spotifyd";
+    rev = "v${version}";
+    sha256 = "0nmi33wvagmasxb59jwhlgdqlh8gfmqgwib16k1wx885qhx9yg30";
+  };
+
+  cargoPatches = [ ./noavxcrypto.patch ];
```

Comment:
I comment with why this was needed could be helpful to future you or other maintainers.